app/plugin_api_test.go

```diff
@@ -220,6 +224,38 @@ func TestPluginAPIGetProfileImage(t *testing.T) {
 	require.Nil(t, data)
 }
 
+func TestPluginAPISetProfileImage(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	api := th.SetupPluginAPI()
+
+	// Create an 128 x 128 image
+	img := image.NewRGBA(image.Rect(0, 0, 128, 128))
+	// Draw a red dot at (2, 3)
+	img.Set(2, 3, color.RGBA{255, 0, 0, 255})
+	buf := new(bytes.Buffer)
+	err := png.Encode(buf, img)
+	dataBytes := buf.Bytes()
+
+	// Set the user profile image
+	data, err := api.SetProfileImage(th.BasicUser.Id, dataBytes)
+	require.Nil(t, err)
+	require.NotEmpty(t, data)
+
+	// Get the user profile image to check
+	imageProfile, err := api.GetProfileImage(th.BasicUser.Id)
+	require.Nil(t, err)
+	require.NotEmpty(t, imageProfile)
+
+	byteReader := bytes.NewReader(imageProfile)
+	img2, _, err2 := image.Decode(byteReader)
+	if err2 != nil {
+		t.Fatal(err)
+	}
```

Comment:
You can just use `require.Nil(t, err2)` here.

Reply:
done

Reply:
thanks for the review @hanzei